### PR TITLE
Add privacy-and-cookies CMS page, remove non-CMS terms-and-conditions…

### PR DESCRIPTION
**Pull request by konradko**
… page

### CODE REVIEW

File: ui/urls.py [hunk 1]
```diff
@@ -49,22 +49,22 @@
...
     url(
         r"^privacy-and-cookies/$",
-        core.views.PrivacyCookiesDomestic.as_view(),
+        core.views.PrivacyCookiesDomesticCMS.as_view(),
         name='privacy-and-cookies'
     ),
     url(
         r"^terms-and-conditions/$",
-        core.views.TermsConditionDomesticViewNegotiator.as_view(),
+        core.views.TermsConditionsDomesticCMS.as_view(),
```
Comment (richtier) [inline ui/urls.py]: no feature flag?

File: core/tests/test_views.py [hunk 2]
```diff
@@ -154,27 +155,26 @@ def test_terms_conditions_cms_disabled(
...
+            'privacy-and-cookies-international',
+            'core/privacy_cookies-international-cms.html'
         ),
     )
 )
-@patch('core.views.cms_client.export_readiness.get_terms_and_conditions_page')
-def test_terms_conditions_cms_enabled(
-    mock_get_t_and_c_page, view, expected_template, client, settings
+@patch('core.views.cms_client.export_readiness.get_privacy_and_cookies_page')
+def test_privacy_cookies_cms(
```
Comment (richtier) [inline core/tests/test_views.py]: it looks like this has replaced the terms and conditions test?

Comment (konradko) [reply core/tests/test_views.py]: t&c is still up `test_terms_conditions_cms`

File: ui/urls.py (hunk 1 above)
Comment (konradko) [reply ui/urls.py]: no need for one - with the first page added it was important to be able to roll back quickly, but now we can just keep replacing.

**Pull request merged**